inspire/config.py

```diff
@@ -230,6 +230,7 @@
     "reportnumber": ["report_number.value"],
     "experiment": ["accelerator_experiment.experiment"],
     "title": ["title.title"],
+    "department_acronym": ["department_acronym"],
```

Comment:
Did you try if this is needed? Usually you need to configure it in cases where the value is in a second level like `accelerator_experiment.experiment` but for first level I thought it worked by default.